Nziranziza/Send-IT
# #162047499 changed folder structure and organized codes

**What does this PR do?**

* change the folder structure
* make codes more organized and easy to read

**Description of the task to be completed?**

* made a server folder and copied routes, test, data, controllers into it
* removed the bin and configure app.js  to start script in package json
* removed public, views folders

**How should this be tested manually**

Once you visit the project repository you will see the server folder and all the changes


**Any background context you want to provide**

No, there isn't

**What are the relevant pivotal tracker stories?**

#162047499
